dao2caom2/telescopes.py

```diff
@@ -908,18 +894,16 @@ def accumulate_blueprint(self, bp):
 
 
 class Dao12MetreImage(Dao12Metre, Imaging):
-    def __init__(self, storage_name, headers, clients, observable, observation, config):
-        super().__init__(storage_name, headers, clients, observable, observation, config)
+    def __init__(self, storage_name, clients, observable, observation, config):
+        super().__init__(storage_name, clients, observable, observation, config)
```

Comment:
Why are these required? Doesn't otherwise use the superclass ctor automatically?

Reply:
Done.